eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
-    expect(getByText("Section two")).toBeVisible();
-    expect(getByText("Section three")).toBeVisible();
-    expect(getByText("Section four")).toBeVisible();
-  });
+    it("should display question pages in 'Current section'", () => {
+      const { getByText } = defaultSetup();
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+      expect(getByText("Question one")).toBeVisible();
+      expect(getByText("Question two")).toBeVisible();
+      expect(getByText("Question three")).toBeVisible();
+      expect(getByText("Question four")).toBeVisible();
+    });
 
-  it("should display 'Other destinations' options", () => {
-    const { getByText, click } = defaultSetup();
-    click(tabTitles.other);
-    expect(getByText(tabTitles.other)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-    expect(getByText(destinationKey[NextPage])).toBeVisible();
-    expect(getByText(destinationKey[EndOfCurrentSection])).toBeVisible();
-    expect(getByText(destinationKey[EndOfQuestionnaire])).toBeVisible();
-  });
+    it("should display sections in 'Later sections'", () => {
+      const { queryByText, getByText, click } = defaultSetup();
+      click(tabTitles.later);
+      expect(getByText(tabTitles.later)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+      expect(queryByText("Section one")).toBeFalsy();
+      expect(getByText("Section two")).toBeVisible();
+      expect(getByText("Section three")).toBeVisible();
+      expect(getByText("Section four")).toBeVisible();
+    });
 
-  it("should display only display 'Current section' and 'Other destinations' tab when missing destinations in the 'Later sections'", () => {});
-
-  it("should be able to change destination tabs with Space or enter", () => {
-    const { getByText, keyPress } = defaultSetup();
-
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-
-    // doesn't change with other keyDown
-    keyPress(tabTitles.later, { key: "Escape", code: "Escape" });
-    expect(getByText(tabTitles.current)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-
-    // changes with Enter
-    keyPress(tabTitles.later, { key: "Enter", code: "Enter" });
-    expect(getByText(tabTitles.later)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-
-    // changes with Space
-    keyPress(tabTitles.other, { key: " ", code: "Space" });
-    expect(getByText(tabTitles.other)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should display 'Other destinations' options", () => {
+      const { getByText, click } = defaultSetup();
+      click(tabTitles.other);
+      expect(getByText(tabTitles.other)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
```

Comment:
The same as above regarding testing the active indicator rather than the styles.